hypervisor/common/hypercall.c

```diff
@@ -150,11 +150,13 @@ int32_t hcall_create_vm(struct acrn_vm *vm, uint64_t param)
 	struct acrn_vm *target_vm = NULL;
 	struct acrn_create_vm cv;
 	struct acrn_vm_config* vm_config = NULL;
+	bool valid;
 
 	(void)memset((void *)&cv, 0U, sizeof(cv));
 	if (copy_from_gpa(vm, &cv, param, sizeof(cv)) == 0) {
 		vm_id = get_vmid_by_uuid(&cv.uuid[0]);
-		if (vm_id < CONFIG_MAX_VM_NUM) {
```

Comment:
we'd better to check vm_id is valid here.

Reply:
both place are checked.   
CI  says "procedure has more than one exit point.", so i delete the changes in create_vm